Maths/JugglerSequence.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Juggler Sequence: https://en.wikipedia.org/wiki/Juggler_sequence
+ * function jugglerSequence
+ * Juggler Sequence is a series of integer number in which the first term starts with a positive integer number n
+ * and the remaining terms are generated from the immediate previous term using the recurrence relation
+ * Produce Juggler Sequence using number n as the first term of the sequence and store in an array
+ * Reference: https://www.geeksforgeeks.org/juggler-sequence/
+ * jugglerSequence(3) // returns [3, 5, 11, 36, 6, 2, 1 ]
+ * isCompositeNumber(9) // returns [9, 27, 140, 11, 36, 6, 2, 1]
+ * isCompositeNumber(15) // returns [15, 58, 7, 18, 4, 2, 1]
+ */
+
+function jugglerSequence (n) {
+  const sequence = []
+  let a = n
+  sequence.push(n)
+  // Calculate terms until last term is not 1
+  while (a !== 1) {
+    let b = 0
+    // Check if previous term is even or odd
+    if (a % 2 === 0) {
+    // calculate next term
+      b = Math.floor(Math.sqrt(a))
+    } else {
+      // for odd previous term calculate next term
+      b = Math.floor(Math.sqrt(a) * Math.sqrt(a) * Math.sqrt(a))
```

Comment:
`sqrt(a)^3 = sqrt(a)^2 * sqrt(a) = a * sqrt(a) = a ^ 1.5` (`a ** 1.5` in JS), please simplify; once you have simplified this, you might simplify the entire `if-else` to a ternary:

`b = (a % 2 === 0) ? Math.sqrt(a) : (a * Math.sqrt(a))`

which you then may simplify to

`b = ((a % 2 === 0) ? 1 : a) * Math.sqrt(a)`

which can alternatively be written as

`b = a ** (a % 2 === 0) ? 0.5 : 1.5)`

which can again be simplified to

`b = a ** ((a % 2) + 0.5)`

Reply:
Still need Math.floor() function to return the largest integer less than or equal to a given number.
Thanks a lot!